manifest.js

```diff
@@ -488,5 +490,8 @@ export default {
         }
       ]
     }
-  ]
+  ],
+  metaData: {
+    maximumAge: 18 /**IN YEARS*/
```

Comment:
```suggestion
    maximumAge: 20 /**IN YEARS*/
```

Using the current formula, to allow 19 year olds to apply this should be 20

Reply:
This makes sense, will update the age in manifest 🙌